KubeArmor/build/compile.sh

```diff
@@ -15,6 +15,10 @@ if [ -f "$FILE" ]; then
     fi
 fi
 
+cd /KubeArmor/BPF/tests
+make clean
+make
```

Comment:
Can we have a quick check by deploying the init container on GKE?
Not sure if it would find linux/bpf.h without the right KRNDIR supplied.

Reply:
@daemon1024 I updated the PR to pass down the correct value of KRNDIR.